qan-api2/main.go

```diff
@@ -106,7 +106,6 @@ func runGRPCServer(ctx context.Context, db *sqlx.DB, mbm *models.MetricsBucket,
 		l.Debug("RPC response latency histogram enabled.")
 		grpc_prometheus.EnableHandlingTimeHistogram()
 	}
-	grpc_prometheus.Register(grpcServer)
```

Comment:
I wonder, is this an intended removal?

Reply:
Yes it is. Should be registered with another one. Two different registrations for same metrics are forbidden with new version. Everything should work properly, but testing will give us confirmation. With middleware usage will be probably different anyway.